contracts/Ink_SpokePool.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+import "@eth-optimism/contracts/libraries/constants/Lib_PredeployAddresses.sol";
+
+import "./Ovm_SpokePool.sol";
+import "./external/interfaces/CCTPInterfaces.sol";
+import { IOpUSDCBridgeAdapter } from "./external/interfaces/IOpUSDCBridgeAdapter.sol";
+
+/**
+ * @notice Ink Spoke pool.
+ * @custom:security-contact bugs@across.to
+ */
+contract Ink_SpokePool is Ovm_SpokePool {
+    using SafeERC20 for IERC20;
+
+    // Address of the custom L2 USDC bridge.
+    address private constant USDC_BRIDGE = 0xbD80b06d3dbD0801132c6689429aC09Ca6D27f82;
```

Comment:
nb. this is the WorldChain bridge address, so we'd probably want to make it obvious that this is not to be used (might set to 0x0?)

Reply:
https://github.com/across-protocol/contracts/pull/808/commits/2399d8eba61cf4dced94aad2c7c191804ea594fe